lib/ProMotion/web/web_screen_module.rb

```diff
@@ -119,13 +119,15 @@ def open_in_safari(inRequest)
 
     # UIWebViewDelegate Methods - Camelcase
     def webView(inWeb, shouldStartLoadWithRequest:inRequest, navigationType:inType)
```

Comment:
Nitpicky, but could you rename these to `in_request` and `in_type` for consistency?